advanced_functionality/multi_model_linear_learner_home_value/linear_learner_multi_model_endpoint_inf_pipeline.ipynb

```diff
@@ -0,0 +1,1302 @@
+{
```

Comment:
<ol><li>The limited access functionality is pretty interesting. It could be hard to understand for customers with less context on AWS permissions model though. Would it be a good idea to add more details on that here?</li><li>If the role does not allow the actions listed, how can customers self-serve to continue with the notebook?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>Added this explanation to the section :</p><p> </p><p>If the role attached to this notebook instance allows invoking SageMaker endpoints, it is able to invoke all models hosted on the MME.&nbsp;Using IAM conditional keys, you can restrict this model invocation access to specific models.&nbsp;To explore this, you will create a new IAM role and IAM policy with conditional key to restrict access to a single model.&nbsp;Assume this new role and verify that only a single target model can be invoked.</p><p> </p><p>Note that to execute this section, the role attached to the notebook instance should allow the following actions :</p><p>&nbsp;&nbsp;"iam:CreateRole",</p><p>&nbsp;&nbsp;"iam:CreatePolicy",</p><p>&nbsp;&nbsp;"iam:AttachRolePolicy",</p><p>&nbsp;&nbsp;"iam:UpdateAssumeRolePolicy"</p><p>&nbsp;&nbsp;&nbsp;</p><p>If this is not the case, please work the Administrator of this AWS account to ensure this.&nbsp;&nbsp;</p>